libparsec/crates/client/src/certificates_ops/add.rs

```diff
@@ -12,7 +12,7 @@ use crate::{
     certificates_ops::store::CertificatesStoreReadExt, event_bus::EventInvalidCertificate,
 };
 
-#[derive(Debug, thiserror::Error)]
+#[derive(Debug, thiserror::Error, PartialEq, Eq)]
```

Comment:
I'd rather avoid to add PartialEq/Eq only for test purpose where we can use `p_assert_matches` instead

Reply:
Done